src/vivarium/framework/randomness/index_map.py

```diff
@@ -215,7 +215,7 @@ def _convert_to_ten_digit_int(self, column: pd.Series) -> pd.Series:
 
         """
         if isinstance(column.iloc[0], datetime.datetime):
-            column = self._clip_to_seconds(column.view(np.int64))
+            column = self._clip_to_seconds(column.astype(np.int64))
```

Comment:
unrelated change to remove [pd.Series.astype DeprecationWarnings](https://pandas.pydata.org/docs/reference/api/pandas.Series.view.html)